### ISSUE TITLE
When trying to open a .rtf file NVDA crashes and I lose all speech

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5001]
**Reported by Brendon22 on 2015-03-26 19:52**
Hello, Today, I tried to open a .rtf. File, NVDA crashes and I lose all speech in NVDA and I got the error sound then the words unknown, then there wasn't any voice going at all. When looking at the log, I see the following:

Invalid NVDAObject: no windowHandle
Stack trace:
  File "comtypes_comobject.pyc", line 127, in call_without_this
  File "_UIAHandler.pyc", line 256, in IUIAutomationPropertyChangedEventHandler_HandlePropertyChangedEvent
  File "NVDAObjects__init__.pyc", line 70, in **call**
IO - speech.speak (05:33:45):
Speaking [L

**Comment by bhavyashah** [comment]
I doubt that this issue even exists at present. @jcsteh @Qchristensen

**Comment by Qchristensen** [comment]
Brendan,  Can you please advise - are you able to use ANY Microsoft office program at all (either loading a document or opening a blank new document) - either Word, Excel, PowerPoint etc?  Also, can you please confirm which version of Windows you are using?

Also if you could please provide a full log when you get this freeze, that could be really helpful please.

The only other thing I was wondering (probably more for @bhavyashah etc)- I know we've seen (other, unrelated to this issue) problems caused by uninstalling office, could this be something like that?  (Are those other issues fixed if you reinstall office?)

**Comment by bhavyashah** [comment]
@Qchristensen No, some issues with regards to 'unknown' I had reported elsewhere with a system I no longer have access to may not be related to MS Office, because I am certain that one or the other version of Office was definitely installed on that laptop. MS Office may have been uninstalled at some point in time and reinstalled, but at the time of testing, MS Office was certainly present (whether installed or reinstalled).
I doubt that Brendon is even following this ticket anymore, as we have lost touch with many ticket reporters in our transition from Trac to Github, and Brendon isn't one I've observed partake in the switch. Thus, unless we have active users who can replicate the issue, I would sincerely suggest closing. @feerrenrut
